go/client/cmd_config_test.go

```diff
@@ -1,354 +0,0 @@
-// Copyright 2015 Keybase, Inc. All rights reserved. Use of
```

Comment:
If I'm wrong about that, I can revisit.  Would probably want to move the tests into systests to test reloading, etc.